### ISSUE TITLE
About the LLM model size used in the paper

### ISSUE THREAD

**Issue by RayYoh** [saidwivedi/InteractVLM#6]
Hi authors, thanks for your great work. I notice that the LLM version used in the code ([here](https://github.com/saidwivedi/InteractVLM/blob/f3c0aaf7d97d216f2f928904dedc99e2fd1e27d6/scripts/run_train.sh#L57)) is 13B, and I am wondering if the results used in the paper are also 13B or a 7B version?

**Comment by saidwivedi** [comment]
@RayYoh thank you for your interest!

Unless specified, all the results in the paper use 13B model. We have an ablation in supplementary paper which shows that there is a minor degradation in performance when using the 7B model.

**Comment by RayYoh** [comment]
> [@RayYoh](https://github.com/RayYoh) thank you for your interest!
> 
> Unless specified, all the results in the paper use 13B model. We have an ablation in supplementary paper which shows that there is a minor degradation in performance when using the 7B model.

Hi @saidwivedi, thanks for your reply. I am interested in the object affordance prediction, and I see only 13B model performance is provided. May I ask whether you have the performance of the 7B model?

**Comment by saidwivedi** [comment]
@RayYoh For the ablation, I only trained the 7B model for human contact prediction but I do not have one for object affordance. You can easily train one by changing the base pretrained version in the [config](https://github.com/saidwivedi/InteractVLM/blob/18ae1bad4579e8fa30b440f4aa2b8f3d450925e8/scripts/run_train.sh#L57) to `xinlai/LISA-7B-v1`.

Due to repeated demands, I plan to release a 7B model for both human and object contact (`hcontact-occontact`). But I can not confirm a date yet.

**Comment by RayYoh** [comment]
Hi @saidwivedi , thanks for your reply. I have tried to train the model. But I get an error in this line [here](https://github.com/saidwivedi/InteractVLM/blob/18ae1bad4579e8fa30b440f4aa2b8f3d450925e8/datasets/ocontact_3d.py#L149C5-L158C18), since cannot find `sam_image`.

**Comment by saidwivedi** [comment]
Hi @RayYoh , You need to generate the rendered images before training. Alternatively, you can use the preprocessed data I recently uploaded. Please check the `fetch_data.sh` script or README for instructions on how to download it.

**Comment by RayYoh** [comment]
Thanks @saidwivedi for providing the preprocessed data. 
But I find that only the Refrigerator in the test set (piad) is provided in the rendered_points_heatmap_1025 folder. 
And during training, many `frontleft` images do not exist.
